docs/octopus-rest-api/octopus-cli/install-autocomplete.md

```diff
@@ -0,0 +1,14 @@
+---
+title: install-autocomplete
+description: Install a shell auto-complete script into your shell profile, if they aren't already there. Supports pwsh, zsh, bash & powershell.
+---
+
+Install a shell auto-complete script into your shell profile, if they aren't already there. Supports pwsh, zsh, bash & powershell.
+
+**install-autocomplete options**
+
+```text
+Please specify the type of shell to install autocompletion for: --shell=XYZ. Valid values are Pwsh, Zsh, Bash and Powershell.
```

Comment:
@jburger - looks like we missed a test - the command doesn't support help properly.

Reply:
@matt-richardson thanks I'll see about another PR to the CLI to fix this up